src/net.cpp

```diff
@@ -53,6 +53,15 @@ static constexpr int DUMP_PEERS_INTERVAL = 15 * 60;
 /** Number of DNS seeds to query when the number of connections is low. */
 static constexpr int DNSSEEDS_TO_QUERY_AT_ONCE = 3;
 
+/** How long to delay before querying DNS seeds
+ */
+static constexpr std::chrono::seconds DNSSEEDS_DELAY_FEW_PEERS{11};
+static constexpr std::chrono::seconds DNSSEEDS_DELAY_MANY_PEERS{300};
```

Comment:
nit: // 5 minutes, also move them to net.h ?

Reply:
They're not needed from other modules, so doesn't make sense to have them in net.h